mlir/lib/Dialect/Vector/Transforms/LowerVectorTransfer.cpp

```diff
@@ -479,7 +483,7 @@ struct TransferReadToVectorLoadLowering
       return rewriter.notifyMatchFailure(read, "out-of-bounds needs mask");
 
     // Create vector load op.
-    Operation *loadOp;
+    Operation *res;
```

Comment:
Why rename this from loadOp?
```suggestion
    Operation *loadOp;
```

Reply:
Because it can be a BroadcastOp. from l.507